Makefile

```diff
@@ -149,111 +110,54 @@ e2e_targets := test-e2e $(e2e_tests)
 .PHONY: test-e2e-setup
 export KIND_CLUSTER := operator-sdk-e2e
 export KUBECONFIG := $(HOME)/.kube/kind-$(KIND_CLUSTER).config
-export KUBEBUILDER_ASSETS := $(PWD)/tools/bin
+export KUBEBUILDER_ASSETS := $(PWD)/$(TOOLS_DIR)
 test-e2e-setup: build
-	./tools/scripts/fetch kind 0.9.0
-	./tools/scripts/fetch envtest 0.6.3
-	./tools/scripts/fetch kubectl ${K8S_VERSION} # Install kubectl AFTER envtest because envtest includes its own kubectl binary
-	[[ "`./tools/bin/kind get clusters`" =~ "$(KIND_CLUSTER)" ]] || ./tools/bin/kind create cluster --image="kindest/node:v$(K8S_VERSION)" --name $(KIND_CLUSTER)
+	$(SCRIPTS_DIR)/fetch kind 0.9.0
+	$(SCRIPTS_DIR)/fetch envtest 0.6.3
+	$(SCRIPTS_DIR)/fetch kubectl $(K8S_VERSION) # Install kubectl AFTER envtest because envtest includes its own kubectl binary
+ifeq (,$(findstring $(KIND_CLUSTER),$(shell $(TOOLS_DIR)/kind get clusters -q)))
```

Comment:
This line is always evaluated for all make targets, so it fails unless `kind` is already installed in `$TOOLS_DIR`. Other than the fact that this change splits this long line, is this syntax better than what's there?

Reply:
I changed the conditional language because it will never execute the conditional code if `$(KIND_CLUSTER)` is not found (output is less verbose), whereas the bash `[[ ]]` line always executes. You're right though, we shouldn't assume tools are installed at all times.